ironfish/src/blockchain/blockchain.ts

```diff
@@ -100,6 +100,8 @@ export class Blockchain {
 
   private _head: BlockHeader | null = null
 
+  private latestCheckpoint: BlockHeader | null = null
```

Comment:
Can you add some tests around this property and when it is set for sanity sake? I don't feel strongly about whether they need to be whole new tests or just added expects to the existing tests, up to you.

Reply:
Added asserts to existing tests for this